samples/C/dynarray.cats

```diff
@@ -1,56 +0,0 @@
-/* ******************************************************************* */
```

Comment:
I couldn't find any similar `C` file with `.cats` extension. @pchaigno 
@arfon Is it required?